pkg/cli/scaffold.go

```diff
@@ -0,0 +1,108 @@
+/*
```

Comment:
Hi @rumstead,
 
We need to create under alpha a dir for the new command. 
I would suggest that to be regen
See a draft about how I thought that we could move achieve this goal: https://github.com/kubernetes-sigs/kubebuilder/pull/3007